LearningMissionLab/Profile.cs

```diff
@@ -36,7 +37,7 @@ public class Profile : IReport
             this._accountId = accountId;
             this._firstName = firstName;
             this._lastName = lastName;
-            this._financialId = financialId;
+            this._financialAid = financialId;
```

Comment:
Profile should have only one field/property for the financial data - FinancialInfo, please add it to the profile and remove all other financial fields like financial aid or financial id.